src/main/java/at/hannibal2/skyhanni/features/misc/CollectionTracker.kt

```diff
@@ -84,16 +89,22 @@ object CollectionTracker {
 
         val foundInternalName = NeuInternalName.fromItemNameOrNull(rawName)
         if (foundInternalName == null) {
-            ChatUtils.userError("Item '$rawName' does not exist!")
-            return
-        }
 
-        val stack = foundInternalName.getItemStackOrNull()
-        if (stack == null) {
-            ChatUtils.userError("Item '$rawName' does not exist!")
-            return
+            val possibleDungeonBossMatches = DungeonFloor.entries.filter { it.bossName.lowercase().contains(rawName) }
```

Comment:
use a function like `DungeonFloor.getByName`, and only return one entry instead of a list of possible entries. then you dont need the addtional checks in line 95